cmd/run.go

```diff
@@ -503,8 +510,10 @@ func executeRun(runFilePath string, apps []runfileconfig.App) (bool, error) {
 		runStates = append(runStates, runState)
 
 		// Metadata API is only available if app has started listening to port, so wait for app to start before calling metadata API.
-		// The PID is put as 0, so as to not kill CLI process when any one of the apps is stopped.
-		_ = putCLIProcessIDInMeta(runState, 0)
+		_ = putCLIProcessIDInMeta(runState, os.Getpid())
+
+		// Update extended metadata with run file path.
+		_ = putRunFilePathInMeta(runState, runFilePath)
```

Comment:
Are we discarding the errors intentionally? If yes, we can also discard the L.H.S. altogether.

Reply:
yes. Actually no need to return an err from the method itself, because we just want to log and continue.